File: 백준/Bronze/16017. Telemarketer or not？/Telemarketer or not？.c
#include <stdio.h>

int main()
{
    int numArr[5], i;

    for (i = 0; i < 4; i++)
        scanf("%d", &numArr[i]);
    
    if ((numArr[0] == 8 || numArr[0] == 9) && (numArr[3] == 8 || numArr[3] == 9) && numArr[1] ==  numArr[2])
        printf("ignore");
    else
        printf("answer");

    return 0;
}

File: 백준/Bronze/16017. Telemarketer or not？/Telemarketer or not？.py
numArr = [int(input()) for i in range(4)]
if (numArr[0] == 8 or numArr[0] == 9) and (numArr[3] == 8 or numArr[3] == 9) and numArr[1] ==  numArr[2]:
    print("ignore")
else:
    print("answer")